ibm_ace/datadog_checks/ibm_ace/config_models/instance.py

```diff
@@ -41,7 +41,7 @@ class Config:
     mq_port: int
     mq_server: str = Field(..., min_length=1)
     mq_user: Optional[str] = Field(None, min_length=1)
-    mqcd_version: Optional[Literal[1, 2, 3, 4, 5, 6, 7, 8, 9]]
+    mqcd_version: Optional[Literal['1', '2', '3', '4', '5', '6', '7', '8', '9']]
```

Comment:
not sure that this change is needed

Reply:
it was autogenerated why I sync'ed the models - and I suspect it's actually what's making the validation fail in the CI. I hope agent integrations help us figure this out :)